apps/react/src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,69 @@
+import { memo, FC } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import {
+  AppBar, Box, Button, Link, Toolbar,
+} from '@mui/material';
+import { AuthActions } from '@js-camp/react/store/auth/dispatchers';
+import { selectUser } from '@js-camp/react/store/auth/selectors';
+
+import { useAppDispatch, useAppSelector } from '../../store';
+
+const AppHeaderComponent: FC = () => {
+
+  const user = useAppSelector(selectUser);
+
+  const dispatch = useAppDispatch();
+
+  const handleUserLogout = () => {
+    dispatch(AuthActions.logoutUser());
+  };
+
```

Comment:
We don't handle anything like logout in UI, we handle user actions, let's improve naming here
```suggestion

  const handleLogoutButtonClick = () => {
    dispatch(AuthActions.logoutUser());
  };

```